### PR TITLE
repl function updated

### PR DESCRIPTION
**Pull request by meta00**
The function allows to
- replicate the experiment by a number of 'times'
OR
- append simulations to the experiment by a named vector 'by.rows'

### CODE REVIEW

File: R/new_functions.R [hunk 1]
```diff
@@ -433,29 +433,55 @@ observation.experiment <- function(x) {
...
+#' @param exp An object of class \code{experiment}.
+#' @param times Number of replicates
+#' @param by.rows A vector of rows to propagate in \code{exp}.
 #'
 #' @return An object of class \code{experiment} that is the replication of the
 #' inputed \code{x} argument.
 #'
 #' @export
-repl <- function(x, n) UseMethod("repl")
+repl <- function(exp, times = NULL, by.rows = NULL) UseMethod("repl")
```
Comment (choisy) [inline R/new_functions.R]: You don't need 2 arguments `times` and `by.rows` here. Please combine them into one single argument that can either be

- a named vector or
- a single unnamed value

Comment (meta00) [reply R/new_functions.R]: Done

File: R/new_functions.R [hunk 2]
```diff
@@ -433,29 +433,53 @@ observation.experiment <- function(x) {
...
+repl.default <- function(exp, times = NULL) "Unknown class"
 
 
 
 
 #' @describeIn repl
 #' @export
-repl.experiment <- function(x, n) {
-  do.call(rbind, lapply(1:n, function(y) x))
+repl.experiment <- function(exp, times = NULL) {
```
Comment (choisy) [inline R/new_functions.R]: Can you please test this function on the following object:
``` r
exp <- load_experiment("sir", system.file("examples", "sir.gaml", package = "rama"))
```
exploring all the possible cases and include this as examples in the documentation of the function.

Comment (meta00) [reply R/new_functions.R]: <ok pls check the new updated

**Comment by choisy** [comment]
Excellent, thanks.

**Pull request closed without merging**